ProjWidget.py

```diff
@@ -0,0 +1,151 @@
+from PyQt5 import QtCore
```

Comment:
Here's one that'll seem nitpicky - but would you consider running all your source code through the formatter `black`? This should go in a follow-up PR so it doesn't obfuscate what you're doing here... 

It'll format everything consistent to our style guide - and in a way that we can _all_ complain the same amount about its choices...

Reply:
Yeah, again, not mine originally.  So some is indented 2, some 3, and some 4.

I'll reformat after everyone is happy with the content.